contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/ContentNode.vue

```diff
@@ -62,7 +62,7 @@
                       <VListTileAction
                         class="action-col px-1 v-list__group__header__append-icon"
                       >
-                        <Icon icon="dropdown" />
+                        <KIcon :icon="open ? 'dropup' : 'dropdown'" :style="{ fontSize: '22px' }" />
```

Comment:
Not a blocker, but I'm wondering if there's a way to make a nice visual transition between these two icons.